weblogic-azure-aks/src/main/arm/createUiDefinition.json

```diff
@@ -18,6 +18,14 @@
                 "type": "Microsoft.Common.Section",
                 "label": "Credentials for WebLogic",
                 "elements": [
+                    {
+                        "name": "listVMSizes",

```

Comment:
Move the REST API query to the **Basics** blade, then the data may be ready when comes to **AKS** blade.